news/2017-11-28-OWC-Round-of-16-recap.md

```diff
@@ -0,0 +1,43 @@
+---
+layout: post
+title: "osu! World Cup 2017 Round of 16 Recap"
+date: 2017-11-28 14:00:00 +0000
+---
+
+16 teams took to the stage over the weekend aiming to make it through to next weekend, and in fact, they all did! While the Round of 16 is the only stage that doesn't see any teams getting knocked out, there is still plenty on the line. 
```

Comment:
Sixteen teams took to the stage last weekend in the trials of the Round of 16! It might be the only round of the tournament where nobody gets knocked out, but the stakes have never been higher with each team sizing one another up for weaknesses and advantage. Here's what went down!